google/cloud/bigtable/data/_async/mutations_batcher.py

```diff
@@ -203,25 +204,18 @@ def __init__(
           - flow_control_max_bytes: Maximum number of inflight bytes.
           - batch_operation_timeout: timeout for each mutate_rows operation, in seconds. If None,
               table default_operation_timeout will be used
-          - batch_per_request_timeout: timeout for each individual request, in seconds. If None,
-              table default_per_request_timeout will be used
+          - batch_attempt_timeout: timeout for each individual request, in seconds. If None,
+              table default_attempt_timeout will be used
```

Comment:
if None, table default mutate_rows_attempt_timeout will be used. If that's also none, table's default operation_timeout will be used.

Reply:
good catch!